app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: None) -> None:

```

Comment:
The parameter `visitor` should have a type annotation instead of `None`. Consider specifying the expected type for `visitor`.